components/events/summer-school/FaqSection.vue

```diff
@@ -212,82 +212,77 @@
   </section>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component } from 'vue-property-decorator'
+<script setup lang="ts">
 import { CtaClickedEventProp } from '~/constants/segment'
 
 interface TrackedLink {
   url: string,
   segment: CtaClickedEventProp
 }
 
-@Component
-export default class FaqSection extends Vue {
-  khanacademy: TrackedLink = {
-    url: 'https://www.khanacademy.org/math/precalculus/x9e81a4f98389efdf:matrices/x9e81a4f98389efdf:multiplying-matrices-by-matrices/v/matrix-multiplication-intro',
-    segment: { cta: 'khanacademy-matrix-multiplication-intro', location: 'faq' }
-  }
+const khanacademy: TrackedLink = {
```

Comment:
suggestion:

```ts
const khanAcademy: TrackedLink = {
```

Reply:
_Camelcased_ in https://github.com/Qiskit/qiskit.org/pull/2935/commits/21f5873a07e445bef4862abeb71c708c4e0e9af0.